backends/bmv2/pna_nic/pnaNic.cpp

```diff
@@ -318,7 +318,7 @@ void ExternConverter_InternetChecksum::convertExternInstance(UNUSED ConversionCo
     auto inst = c->to<IR::Declaration_Instance>();
     cstring name = inst->controlPlaneName();
     auto trim = inst->controlPlaneName().find(".");
-    auto block = inst->controlPlaneName().trim(trim);
+    auto block = name.before(trim);
```

Comment:
Why `name` instead of `inst->controlPlaneName()`?

Reply:
Good qn. I've replaced with `inst->controlPlaneName()`. (Although arguably, I could have made line 320 use `name` instead to be consistent.